### PR TITLE
Add Meet Our Contributors link

### PR DESCRIPTION
**Pull request by ignertic**

### CODE REVIEW

File: contributors/guide/README.md [hunk 1]
```diff
@@ -273,7 +273,7 @@ You may also contact Paris Pittman via direct message on Kubernetes Slack (@pari
 ## Mentorship
 
 Please learn about our mentoring initiatives [here](http://git.k8s.io/community/mentoring/README.md).
-
+Feel free to ask us anything view our [Meet Our Contributors](https://github.com/kubernetes/community/blob/master/mentoring/meet-our-contributors.md) to connect with us.
```
Comment (nikhita) [inline contributors/guide/README.md]: Feel free to ask us anything _during_ our Meet Our Contributors?

**Comment by tpepper** [comment]
Two small concerns here:

1) It looks like @nikhita's comment was marked resolved, but the commit still appears to be missing the "during" word and thus reads strangely.  I'm not sure if the new magical GitHub in place editing did actually accept and resolve the change and it will eventually show up once merged, or not?

2) I don't want to block a first time contribution on style, but do want to point @ignertic at https://chris.beams.io/posts/git-commit/  for guidance on well formed commit messages.  If you can easily amend the commit message and summary to be more like:
```
contrib guide: update link

Add a link to the "Meet Our Contributors" information under the mentoring section of the contributor guide.
```
That is subtly different in ways that make git log output more informative both to a reviewer today and to future people who may be trying to understand past changes in the revision history.

**Comment by ignertic** [comment]
@tpepper thanks for the tip, still new to this whole open source

**Comment by tpepper** [comment]
> @tpepper thanks for the tip, still new to this whole open source

No problem and if you need any direct git assistance on the mechanics of such fix ups, feel free to ping me on k8s slack with the same @ name as here on GH!

**Comment by parispittman** [comment]
/lgtm
/approve

thanks for the contribution!

**Pull request merged**